pydm/widgets/drawing.py

```diff
@@ -1155,7 +1155,11 @@ def isfloat(value):
         for i, pt in enumerate(value, start=1):
             point = pt
             if isinstance(point, str):
-                point = list(map(float, point.split(",")))
+                try:
+                    point = list(map(float, point.split(",")))
```

Comment:
```suggestion
                    point = ast.literal_eval(point)
```
Because
```
In [3]: ast.literal_eval("1, 2, 3")
Out[3]: (1, 2, 3)
```

Reply:
@klauer Nice use of `ast` package here.  I'll build that in, it makes the parser much cleaner.